gendox-core-api/src/main/java/dev/ctrlspace/gendox/gendoxcoreapi/services/UserOrganizationService.java

```diff
@@ -28,4 +53,40 @@ public List<UserOrganization> getAll(UserOrganizationCriteria criteria, Pageable
         return userOrganizationRepository.findAll(UserOrganizationPredicate.build(criteria), pageable).toList();
 
     }
+
+    public void setAdminRoleForOrganizationsOwner(Organization organization) {
+        UserOrganization userOrganization = new UserOrganization();
+        Instant now = Instant.now();
+
+        // user
+        Authentication authentication = SecurityContextHolder.getContext().getAuthentication();
+        JwtDTO jwtDTO = jwtUtils.toJwtDTO((Jwt)authentication.getPrincipal());
+        String userIdString = jwtDTO.getUserId(); // Assuming you have the UUID as a string
+        UUID userId = UUID.fromString(userIdString);
+        User user = userRepository.getById(userId);
+
+        // role
+        Type role = typeRepository.getById(3L);
```

Comment:
In different environments, the ROLE_ADMIN can have different ID. All type selections should be made by type name, or by category-name combination.

I propose the creation of: 
```java
    typeRepository.getByTypeCategoryAndName("ORGANIZATION_ROLE_TYPE", "ROLE_ADMIN")
```